en/micro-integrator/docs/develop/hot-deployment.md

```diff
@@ -2,19 +2,19 @@
 
 Hot deployment is the process of dynamically deploying your synapse artifacts (XML), dataservices(DBS), carbon applications (CAR), etc., in your Micro Integrator. That is, it is not required to restart the server for the artifact deployment to be effective.
 
-Hot deployment is useful for testing your integration artifacts **in a VM environment**. Because you don't have to restart the server every time you deploy an artifact, the testing time will be shorter and efficient.
+Hot deployment is useful for testing the integration artifacts **in a VM environment**. With hot deployment, no 
```

Comment:
"Hot deployment is useful for testing the integration artifacts **in a VM environment**. With hot deployment, you are not required to restart the server each time you deploy an artifact. Therefore, testing time will be shorter and efficient. Hence, this is enabled by default for a VM distribution. This is disabled in the base Micro Integrator Docker image to honor the immutable nature in the container world."

When we say 'enabled by default for a VM distribution', we are saying it is enabled in the Micro Integrator server that we download from the website, yeah? 
- Should we also explain the connection between immutability in container environments and hot deployment? Or is - it understood that, if the environment is immutable, hot deployment should not apply?

Reply:
Yeah we are saying that it is enabled in the distribution we download from the site. 

Since at the very beginning we say as , "Hot deployment is the process of dynamically deploying your synapse artifacts" , it already caters the need of explaining the relation between immutability and hot deployment. wdyt